### DISCUSSION TITLE
WebView2 with C++ on Xbox

### DISCUSSION THREAD

**Discussion by AshleyScirra** [MicrosoftEdge/WebView2Feedback#4121] [Q&A] [answered]
It looks like the current support for WebView2 on Xbox is based on WinUI 2 (UWP) with C#. We have an existing codebase using C++ with WebView2 that we'd like to move over to Xbox. Is there any support for using C++ on Xbox with WebView2, or any plans to add it in future?

**Comment by la-thorn** [comment, marked as answer]
You can use WebView2 on Xbox in a C++ UWP application as well! For an example, you can see this sample application:
https://github.com/microsoft/Media-App-Samples-for-Xbox/tree/main/WebView2/cpp/JavaScriptVideoSample

**Reply by AshleyScirra** [reply]
That looks like a good example of what I'm after, thanks!
